### DISCUSSION TITLE
Support for replace lists and objects in Json Handler

### DISCUSSION THREAD

**Discussion by BeppeTW** [thoughtworks/templify#16] [General]
In the CookieCutter + Backstage world, must Json handler support objects and lists? 

What kind of replaces in projects or templates does this will support?

**Comment by Luiz-Micci** [comment]
I believe not in the first version, we should focus on something simple. As time goes on and as needs appears, we can iterate on the JSON handler.

Do you have any idea or scenario in which this might occur? Is there any recollection of such a need?

**Comment by BeppeTW** [comment]
Need votes
